docs/content/manual/manual.md

```diff
@@ -0,0 +1,5424 @@
+true
+~~~
+
+~~~
+.[] == 1
+[1, 1.0, "1", "banana"]
+true
+true
+false
+false
+~~~
+
+:::
+
+## Ordering: `>`, `>=`, `<=`, `<` {#ordering}
+
+The ordering operators `>`, `>=`, `<=`, `<` return whether
+their left argument is greater than, greater than or equal
+to, less than or equal to or less than their right argument
+(respectively).
+
+Values are ordered as follows, by increasing order:
+
+* `null`
+* `false`
+* `true`
+* numbers
+* strings, in alphabetical order (by unicode codepoint value)
+* arrays, in lexical order
+* objects
+
+The ordering for objects is a little complex: first they're
+compared by comparing their sets of keys (as arrays in
+sorted order), and if their keys are equal then the values
+are compared key by key.
+
+::: Examples
+
+~~~
+. < 5
+2
+true
+~~~
+
+:::
+
+
+# Boolean filters
+
+Every value can be converted to a boolean --- in particular,
+the values `false` and `null` have the _boolean value_ `false`,
+all other values have the boolean value `true`.
+This section describes several filters that analyze the boolean value of values.
+
+You can negate the boolean value of a value with the builtin function [`not`](#not).
+It is called as a filter to which things can be piped
+rather than with special syntax, as in `.foo and .bar | not`.
+
+## if-then-else-end {#if-then-else}
+
+The filter `if i then t else e end`
+runs `t` when `i` returns an output with boolean value `true`, otherwise, it
+runs `e`.
+
+Given three filters `i`, `t`, and `e`,
+the expression `if i then t else e end` runs `i` on its input.
+For every value `y` that is output by `i`,
+if `y` has the boolean value `true`
+(that means, if `y` is neither `false` nor `null`),
+the output of `t` on the original input is produced, else
+the output of `e` on the original input is produced.
+
+::: Note
+Checking for false or null is a simpler notion of
+"truthiness" than is found in JavaScript or Python, but it
+means that you'll sometimes have to be more explicit about
+the condition you want.  You can't test whether, e.g. a
+string is empty using `if .name then A else B end`; you'll
+need something like `if .name == "" then A else B end` instead.
+:::
+
+More cases can be added to an if using `elif A then B` syntax.
+
+`if A then B end` is shorthand for `if A then B else .  end`.
+That is, the `else` branch is optional, and if absent, it is the same as `.`.
+This also applies to `elif` with absent ending `else` branch.
+
+::: Examples
+
+~~~
+if . == 0 then "zero" elif . == 1 then "one" else "many" end
+2
+"many"
+~~~
+
+~~~
+.[] | if . then ., .+1 else . end
+[false, 1, null]
+false
+1
+2
+null
+~~~
+
+:::
+
+## `and`, `or`
+
+The filter `f and g` returns true if both `f` and `g` return an output with boolean value `true`.
+The filter `f or g` returns true if either `f` or `g` return an output with boolean value `true`.
+Otherwise, both filters return `false`.
+
+Given two filters `f` and `g`,
+their conjunction `f and g` and
+their disjunction `f  or g`
```

Comment:
I actually did this on purpose to make the symmetry more visible in the source, but I'll remove it anyway because it is not pretty in the rendered document.